docker/run.sh

```diff
@@ -14,6 +14,7 @@ function setup() {
 
     scripts/uninstall-requirements.sh
     pip install -r requirements/test-requirements.txt
+    hash -d pip
```

Comment:
Huh interesting. What does this do?

Reply:
It removes `/vendor/bin/pip` from bash's command cache. This is necessary if the command path changes, as is the case when pip is upgraded on the line above.